README.md

```diff
@@ -2,7 +2,7 @@
 
 [![status](https://img.shields.io/badge/status-stable-blue.svg)](https://github.com/tauri-apps/tauri/tree/dev)
 [![License](https://img.shields.io/badge/License-MIT%20or%20Apache%202-green.svg)](https://opencollective.com/tauri)
-[![test library](https://img.shields.io/github/workflow/status/tauri-apps/tauri/test%20library?label=test%20library)](https://github.com/tauri-apps/tauri/actions?query=workflow%3A%22test+library%22)
+[![test library](https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/test-core.yml?label=test%20library)](https://github.com/tauri-apps/tauri/actions/workflows/test-core.yml)
```

Comment:
```suggestion
[![test core](https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/test-core.yml?label=test%20core)](https://github.com/tauri-apps/tauri/actions/workflows/test-core.yml)
```

and same for all other READMEs

Reply:
This was already on the description above. Then I will change it to core.

> Also note that test-library was changed to test-core but not sure whether I also change it.

Reply:
my bad, I didn't notice it, thanks for your contribution.